appletree/plugins/detector.py

```diff
@@ -18,15 +18,53 @@
         help="S1 light collection efficiency correction",
     ),
 )
-class S1Correction(Plugin):
+class S1CorrectionTrue(Plugin):
+    depends_on = ["x", "y", "z"]
+    provides = ["s1_correction_true"]
+
+    @partial(jit, static_argnums=(0,))
+    def simulate(self, key, parameters, x, y, z):
+        pos_true = jnp.stack([x, y, z]).T
+        s1_correction_true = self.s1_correction.apply(pos_true)
+        return key, s1_correction_true
+
+
+@export
+@takes_config(
+    Map(
+        name="s1_correction",
```

Comment:
We should not set the same config names of S1 correction for `S1CorrectionTrue` and `S1CorrectionRec`.

Reply:
I think it should be fine? https://github.com/XENONnT/appletree/blob/6a942fba7977a9ec6e3d717e85b49d6743ebefba/appletree/config.py#L150-L155